lib/iris/fileformats/netcdf.py

```diff
@@ -2246,6 +2246,10 @@ def is_valid_packspec(p):
 
         conventions = CF_CONVENTIONS_VERSION
 
+        # Prioritise cube attributes conventions over the default.
+        if 'Conventions' in cube.attributes:
+            conventions = cube.attributes['Conventions']
```

Comment:
@morwenna-01 No worries ... I think that I can resolve this from my side. Let me try ...